dune/xt/la/container/common.hh

```diff
@@ -287,9 +303,13 @@ protected:
 public:
   inline ScalarType& operator[](const size_t ii)
   {
+    ensure_uniqueness();
+    unshareable_ = true;
     return get_entry_ref(ii);
   }
 
+  // TODO: Need to ensure uniqueness() and unshareable_ = true also in this method, but that would imply making
+  // ensure_uniqueness const and all members mutable
   inline const ScalarType& operator[](const size_t ii) const
```

Comment:
We could also think about changing the interface of the const `operator[]` to always return a copy, though I have no feeling for the impact...
```c++
   inline const ScalarType operator[](const size_t ii) const
```